### PR TITLE
chore: remove docs deploy on surge

### PR DESCRIPTION
**Pull request by mainframev**
Storybook: https://orbit-chore-remove-docs-deploy.surge.sh

### CODE REVIEW

**Review by silvenon — DISMISSED** [review]

**Comment by mainframev** [comment]
@silvenon any idea why it fails? seems like it can't apply artifact 🤔

File: .github/workflows/test_and_deploy.yml [hunk 1]
```diff
@@ -48,15 +37,13 @@ jobs:
       - name: Build
         env:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
-        run: npx lerna run build
+        run: npx lerna run --ignore orbit.kiwi build
```
Comment (CollierCZ) [inline .github/workflows/test_and_deploy.yml:40]: This does not leave out the docs build. I think using `@kiwicom/orbit.kiwi` would work, but haven't checked it.

File: .github/workflows/test_and_deploy.yml [hunk 2]
```diff
@@ -71,8 +58,7 @@ jobs:
           yarn prettier:test
 
       - name: Test
-        run: |
-          yarn test-ci
+        run: yarn test-ci
 
       - name: Check docs links
```
Comment (CollierCZ) [inline .github/workflows/test_and_deploy.yml:63]: This step requires a build of the docs. Removing the build means that the check isn't done.

**Review by CollierCZ — CHANGES_REQUESTED** [review]

**Comment by CollierCZ** [comment]
> @silvenon any idea why it fails? seems like it can't apply artifact 🤔

I think it's because the artifact is being saved as a top-level folder, so that's how it's extracted. Before, when it was saving two folders, it saved them in separate folders that were unpacked correctly. Now it's unpacking to the root instead of `packages/orbit-components/.out`.

You have to use the path option for the step: https://github.com/actions/download-artifact#download-a-single-artifact

**Comment by mainframev** [comment]
> > @silvenon any idea why it fails? seems like it can't apply artifact 🤔
> 
> I think it's because the artifact is being saved as a top-level folder, so that's how it's extracted. Before, when it was saving two folders, it saved them in separate folders that were unpacked correctly. Now it's unpacking to the root instead of `packages/orbit-components/.out`.
> 
> You have to use the path option for the step: https://github.com/actions/download-artifact#download-a-single-artifact

Ah, yes, look like it works. Did not notice that little thing, thanks a lot :)

**Review by silvenon — APPROVED** [review]

**Pull request merged**